Buildings/Controls/OBC/FDE/DOAS/SupplyFanController.mo

```diff
@@ -0,0 +1,224 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block SupplyFanController
+  "This block manages start, stop, status, and speed of the supply fan."
+
+ parameter Real minDDSPset(
+   min=0,
+   final unit="Pa",
+   final quantity="PressureDifference")=125
+    "Minimum down duct static pressure reset value"
+      annotation (Dialog(group="DDSP range"));
+ parameter Real maxDDSPset(
+   min=0,
+   final unit="Pa",
+   final quantity="PressureDifference")=500
+    "Maximum down duct static pressure reset value"
+      annotation (Dialog(group="DDSP range"));
+  parameter Real cvDDSPset(
+   min=0,
+   final unit="Pa",
+   final quantity="PressureDifference")=250
+    "Constant volume down duct static pressure set point";
+ parameter Real damSet(
+   min=0,
+   max=1,
+   final unit="1")=0.9
+   "DDSP terminal damper percent open set point";
+ parameter Boolean vvUnit = true
+   "Set true when unit serves variable volume system.";
+
+  // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occ
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{-142,60},{-102,100}}),
+       iconTransformation(extent={{-140,50},{-100,90}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput mostOpenDam
+    "Most open damper position from all terminal units served."
+      annotation (Placement(transformation(extent={{-142,-24},{-102,16}}),
+        iconTransformation(extent={{-140,14},{-100,54}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput supFanStatus
+    "True when supply fan is proven on."
+      annotation (Placement(transformation(extent={{-142,22},{-102,62}}),
+        iconTransformation(extent={{-140,-56},{-100,-16}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput DDSP
+    "Down duct static pressure measurement."
+      annotation (Placement(transformation(extent={{-142,-106},{-102,-66}}),
+        iconTransformation(extent={{-140,-92},{-100,-52}})));
+
+  // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput supFanStart
+   "Command supply fan to start when true."
+     annotation (Placement(transformation(extent={{102,60},{142,100}}),
+       iconTransformation(extent={{100,32},{140,72}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput supFanProof
+    "True when supply fan is proven on"
+      annotation (Placement(transformation(extent={{102,22},{142,62}}),
+        iconTransformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput supFanSpeed
```

Comment:
`supFanSpeed` should be zero, when the `supFanStatus` is false. This do not need to assign through PID controller. Please reimplement this part:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/23491921/93403128-7b15b300-f83b-11ea-9a33-32a9d8a98291.png">

Reply:
This stems from real world applications where the fan speed signal going to the VFD is ignored when the VFD is not enabled to run. Some VFD/motor combinations have a minimum operating frequency which must be maintained when the motor is energized and is typically around 20Hz. When a current switch is used to prove fan operation there is sometimes a minimum run speed required before the current switch will prove operation requiring the minimum speed signal to be something other than 0 prior to a start command. In short, zeroing the set point is common practice in this type of HVAC application.